apps/mochi-web/constants/index.ts

```diff
@@ -43,6 +43,10 @@ export const PAGES = {
     path: '/privacy',
     title: 'Privacy Policty',
   },
+  EXPLORE: {
+    path: '/explore',
```

Comment:
Can you define the route in `constants/routes.ts` instead? For the SEO part, it is not always up to us to the pattern in one place if the page data is dynamic. I would suggest internally defining the ﻿<SEO /> component on each page instead of incompletely centralizing it in one place.